test/unit/provision/synchronousButtonProvision.json

```diff
@@ -28,6 +28,18 @@
           "name": "op_result",
           "type": "string"
         }
+      ],
+      "static_attributes": [
+        {
+          "name": "interaction_type",
+          "type": "string",
+          "value": "synchronous"
+        },
+        {
+          "name": "service_id",
+          "type": "string",
+          "value": "smartcity"
+        }
```

Comment:
Yes, they are. But it doesn't matter really for the unit test. The only thing that has to be tested is that this attributes end up added to the Context Broker queries. The particular attributes to be sent are transparent to the IOTA.